### PR TITLE
Docs: Add dashboard list

### PR DESCRIPTION
**Pull request by oddlittlebird**
This was pretty straightforward except for the **Folder** field. That field was not covered in existing documentation, so please make sure the definition I entered is correct.

### CODE REVIEW

File: docs/sources/panels/visualizations/dashboard-list-panel.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### Options
+
+- **Starred -** Display starred dashboards in alphabetical order.
+- **Recently viewed -** Display recently viewed dashboards in alphabetical order.
+- **Search -** Display dashboards by search query or tags. Requires you to enter at least one value in **Query** or **Tags**.
+- **Show headings -** The chosen list selection (Starred, Recently viewed, Search) is shown as a heading.
+- **Max items -** Sets the maximum number of items to list.
+
+Search
```
Comment (daniellee) [inline docs/sources/panels/visualizations/dashboard-list-panel.md:34]: I would just add that the below options are used in combination with the above search option. They won't work unless you have selected search first (sort of strange UI)

**Review by daniellee — APPROVED** [review]
Just one nitpick

File: docs/sources/panels/visualizations/dashboard-list-panel.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Use these options to refine your visualization.
+
+### Options
+
+- **Starred -** Display starred dashboards in alphabetical order.
+- **Recently viewed -** Display recently viewed dashboards in alphabetical order.
+- **Search -** Display dashboards by search query or tags. Requires you to enter at least one value in **Query** or **Tags**.
+- **Show headings -** The chosen list selection (Starred, Recently viewed, Search) is shown as a heading.
+- **Max items -** Sets the maximum number of items to list.
```
Comment (dprokop) [inline docs/sources/panels/visualizations/dashboard-list-panel.md:32]: I would add that the max items are per section (starred, recent, search). So setting i.e. 10 will result in up to 30 elements on the list

**Review by dprokop — APPROVED** [review]
One nit :)

**Pull request merged**